src/containers/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
Even though this is an import from a library, it is also styling and should be in it's own block of imports